R/interface.R

```diff
@@ -289,9 +289,16 @@ sourceExamples <- function(file){
 rcppoctave <- function(..., plugin = 'RcppOctave'){
 	if( !requireNamespace('inline') )
 		stop("Package 'inline' is required to define RcppOctave functions.")
-	#qlibrary(inline)
-	#rcpp(..., plugin = plugin)
-	cxxfunction(..., plugin = plugin)
+	#qlibrary('inline')
```

Comment:
`qlibrary` from `qtbase` is a heavy dependency. The code was working for me without it. Is it required?